app-template/config-template.xml

```diff
@@ -51,14 +47,17 @@
     <plugin name="cordova-plugin-android-fingerprint-auth" spec="1.4.3" />
     <plugin name="cordova-plugin-device" spec="2.0.1" />
     <plugin name="cordova-plugin-globalization" spec="1.0.9" />
-    <plugin name="cordova-plugin-qrscanner" spec="2.5.0" />
+    <plugin name="cordova-plugin-qrscanner" spec="2.6.0" />
+    <plugin name="cordova-plugin-add-swift-support" spec="1.7.2" />
     <plugin name="cordova-plugin-splashscreen" spec="5.0.2" />
     <plugin name="cordova-plugin-statusbar" spec="2.4.2" />
-    <plugin name="cordova-plugin-inappbrowser" spec="2.0.2" />
-    <plugin name="cordova-plugin-touch-id" spec="3.3.1" />
+    <plugin name="cordova-plugin-inappbrowser" spec="3.0.0" />
+    <plugin name="cordova-plugin-touch-id" spec="3.4.0">
+      <variable name="FACEID_USAGE_DESCRIPTION" value=" " />
+    </plugin>
     <plugin name="cordova-plugin-whitelist" spec="1.3.3" />
-    <plugin name="cordova-plugin-x-socialsharing" spec="5.4.1" />
-    <plugin name="cordova-plugin-x-toast" spec="2.6.0" />
+    <plugin name="cordova-plugin-x-socialsharing" spec="5.4.3" />
```

Comment:
lets not upgrade plugin ins unless there is a security issue.

x-socialsharing // x-toast // qr-scanner

Reply:
+ inappbrowser

Reply:
ok

Reply:
qr-scanner is ok. Version number on package.json was 2.6.0 for long time (maybe we forget to change it on config.xml). Previous version may break the building process with cordova 8: https://github.com/bitpay/cordova-plugin-qrscanner/issues/144#issuecomment-389976578